Makefile

```diff
@@ -225,6 +225,9 @@ check:
 # Ensure we build on linux 386 for Trivy:
 #	gh release view -R aquasecurity/trivy --json assets --jq 'first(.assets[] | select(.name| test("Linux-32bit.*tar.gz")) | {url, downloadCount})'
 	@GOARCH=386 GOOS=linux go build ./...
+# Ensure we build on FreeBSD 386 for Trivy:
+#	gh release view -R aquasecurity/trivy --json assets --jq 'first(.assets[] | select(.name| test("FreeBSD-32bit.*tar.gz")) | {url, downloadCount})'
+	@GOARCH=386 GOOS=freebsd go build ./...
```

Comment:
oh no this should be amd64

Reply:
yep actually they build also on 386 but more important is amd64 agree

Reply:
as amd64 is  the original issue why we have started testing https://github.com/tetratelabs/wazero/issues/607

Reply:
cool. and verified the Makefile comment, also.

```bash
$ gh release view -R aquasecurity/trivy --json assets --jq 'first(.assets[] | select(.name| test("FreeBSD-64bit.*tar.gz")) | {url, downloadCount})'
{"downloadCount":13,"url":"https://github.com/aquasecurity/trivy/releases/download/v0.36.0/trivy_0.36.0_FreeBSD-64bit.tar.gz"}
```